python/distributed-ucxx/distributed_ucxx/tests/test_ucxx.py

```diff
@@ -356,7 +356,7 @@ async def test_cuda_context(
                     )
 
 
-@gen_test()
+@gen_test(timeout=60)
```

Comment:
With some local experimentation it looks like teardown failures are in essence a `gen_test` timeout, but it doesn't show up as such. I tried increasing the timeout here but one of the tests nevertheless failed. Locally though I see this test completing in under 10 seconds or in 60 seconds with this change even though it ultimately passes after the full 60 seconds. With that I'm trying to understand whether this is a problem with UCXX or if this is something with the Distributed test suite (e.g., `gen_test`) as I would expect it to fail every time if it times out.